src/main/java/com/titanic/fork/web/controller/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package com.titanic.fork.web.controller;
+
+import com.titanic.fork.service.account.LoginService;
+import com.titanic.fork.web.dto.request.account.NewPasswordRequest;
+import com.titanic.fork.web.dto.request.account.ValidateNameAndPasswordRequest;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.NoResultException;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("account")
+public class LoginController {
+
+    private final LoginService loginService;
+
+    @ApiOperation(value = "비밀번호 수정을 위한 인증API")
+    @PostMapping("find")
+    public ResponseEntity<Void> validateNameAndPassword(@RequestBody ValidateNameAndPasswordRequest
+                                                                validateNameAndPasswordRequest) {
+        return loginService.validateNameAndPassword(validateNameAndPasswordRequest);
+    }
```

Comment:
보통 url에선 명사를 위주로 쓰며, find라는 찾음(명사)를 쓰기보단 행위에 대상을 적는게 Restful한 URL 이라고 생각해요. 그리고 validate 유효한 값인지 판단하는 거니 Json으로 true, false를 담아주는건 어떨까요?
```suggestion
    @ApiOperation(value = "비밀번호 수정을 위한 인증API")
    @PostMapping("/user)
    public ResponseEntity<Boolean> validateNameAndPassword(@RequestBody ValidateNameAndPasswordRequest
                                                                validateNameAndPasswordRequest) {
        return loginService.validateNameAndPassword(validateNameAndPasswordRequest);
    }
```

Reply:
RestFul에 대해선 막연히 알고 있었는데 다시 한번 공부가 필요한 시점같네요. 

주신 피드백은 한번 고려해보겠습니다.!